drivers/sensor/adxl345/adxl345.h

```diff
@@ -52,8 +50,7 @@ struct adxl345_sample {
 };
 
 struct adxl345_dev_config {
-	const char *i2c_master_name;
-	uint16_t i2c_addr;
+	const struct i2c_dt_spec i2c;
```

Comment:
```suggestion
	struct i2c_dt_spec i2c;
```

Reply:
> Suggested change 
> 	const struct i2c_dt_spec i2c;
> 	struct i2c_dt_spec i2c;

@galak and @gmarull, you seem to disagree on this, which one should we go with?